packages/firebase_ml_vision/example/lib/main.dart

```diff
@@ -2,32 +2,154 @@
 // Use of this source code is governed by a BSD-style license that can be
 // found in the LICENSE file.
 
+import 'dart:async';
+import 'dart:io';
+
 import 'package:flutter/material.dart';
+import 'package:image_picker/image_picker.dart';
+import 'package:firebase_ml_vision/firebase_ml_vision.dart';
 
-void main() => runApp(new MyApp());
+void main() => runApp(new MaterialApp(home: _MyHomePage()));
 
-class MyApp extends StatefulWidget {
+class _MyHomePage extends StatefulWidget {
   @override
-  _MyAppState createState() => new _MyAppState();
+  _MyHomePageState createState() => new _MyHomePageState();
 }
 
-class _MyAppState extends State<MyApp> {
-  @override
-  void initState() {
-    super.initState();
+class _MyHomePageState extends State<_MyHomePage> {
+  File _imageFile;
+  Size _imageSize;
+  List<TextBlock> _textLocations;
+
+  Future<void> _getImage() async {
+    setState(() {
+      _imageFile = null;
+      _imageSize = null;
+      _textLocations = null;
+    });
+
+    final File imageFile =
+        await ImagePicker.pickImage(source: ImageSource.camera);
```

Comment:
Could we use ImageSource.gallery so when using an emulator the sample is just as useful.